cypress/e2e/models/migration/applicationinventory/analysis.ts

```diff
@@ -355,16 +355,8 @@ export class Analysis extends Application {
     }
 
     openReport() {
-        this.selectApplicationRow();
-        cy.get(rightSideMenu, { timeout: 15 * SEC }).within(() => {
-            clickTab("Reports");
-            cy.contains("button", "View analysis", { timeout: 15 * SEC })
-                .then(($a) => {
-                    // Removing target from html so that report opens in same tab
-                    $a.attr("target", "_self");
-                })
-                .click();
-        });
+        // TODO: Update once the new reports feature is implemented
```

Comment:
we don't have any way to open legacy reports ?
Just making sure that we won't need this code at all

Reply:
Not right now, in case we need it, we can get it again using git.